### ISSUE TITLE
Some apps are not showing up

### ISSUE THREAD

**Issue by onbjerg** [1Hive/apiary#86]
**Describe the bug**

A bunch of apps installed in `yachtsocial.aragonid.eth` are not showing up on the website.

Apps that are not showing up:

- Uniswap
- Compound
- ENS

**To Reproduce**

N/A

**Expected behavior**

The apps should have been indexed on the website by the worker before the DAO was created.

**Comment by lkngtn** [comment]
Apps in this org also do not appear to have been caught: https://mainnet.aragon.org/#/0xa1213ff38A8a450f47D4d6a737924A0e3Af3603a/organization/

**Comment by onbjerg** [comment]
Repositories are not caught because they don't really have an app ID, so they're a special case. We'll index them eventually since it would be nice to be able to filter apps based on the registry they were published on, as registries are effectively a way to curate apps (e.g. apps in the official registry are inherently more trusted than the ones in the open registry)

**Comment by onbjerg** [comment]
https://mainnet.aragon.org/#/0x57EBE61f5f8303AD944136b293C1836B3803b4c0/0xa2b6218673bac129127be86106a9d5d205814edc/

Also apps missing there I think (tipping, subscribe, harberger)

**Comment by onbjerg** [comment]
I'm going to do some analysis to see if the apps have anything in common.

Currently the list of apps not showing up are:

**0x57EBE61f5f8303AD944136b293C1836B3803b4c0**

- AirDropduo ([manifest](https://ipfs.eth.aragon.network/ipfs/Qmafjwcmg1ZuZJ9Zdd9R8tbqM2MJat4etoVmke76HXHaR1/manifest.json), [artifact](https://ipfs.eth.aragon.network/ipfs/Qmafjwcmg1ZuZJ9Zdd9R8tbqM2MJat4etoVmke76HXHaR1/artifact.json)): 0xa2b6218673bac129127be86106a9d5d205814edc
- Challenge ([manifest](https://ipfs.eth.aragon.network/ipfs/QmeMRaYhWJkvpumTjN7CQj3ucCYMTW7xeGpcANX28tNGqP/manifest.json), [artifact](https://ipfs.eth.aragon.network/ipfs/QmeMRaYhWJkvpumTjN7CQj3ucCYMTW7xeGpcANX28tNGqP/artifact.json)): 0x5f1207622d65e6225fa6be7045ac0319fb598508
- Harberger ([manifest](https://ipfs.eth.aragon.network/ipfs/QmPEixCGrRjNF2TjGknmLnhswnEZqwsxYtRz9tAXCPgjDe/manifest.json), [artifact](https://ipfs.eth.aragon.network/ipfs/QmPEixCGrRjNF2TjGknmLnhswnEZqwsxYtRz9tAXCPgjDe/artifact.json)): 0xc6cfc6a31e516d1622b80c0864b16f665712f89e
- Subscribe ([manifest](https://ipfs.eth.aragon.network/ipfs/QmW2BsYDuF3skrTCypjGJztxPFmD6AD81gEKcyQDtSuAdH/manifest.json), [artifact](https://ipfs.eth.aragon.network/ipfs/QmW2BsYDuF3skrTCypjGJztxPFmD6AD81gEKcyQDtSuAdH/artifact.json)): 0xd1dc1a5b56ea321a921c74f8307153a58b1efa4d
- Tipping ([manifest](https://ipfs.eth.aragon.network/ipfs/QmTNfX2XJ2Gqjd2jQvE5hzFuXAK1Can4x8V78ytF6xyZfm/manifest.json), [artifact](https://ipfs.eth.aragon.network/ipfs/QmTNfX2XJ2Gqjd2jQvE5hzFuXAK1Can4x8V78ytF6xyZfm/artifact.json)): 0xe3b8ff1ead55bc511127cfe90f5cedbb90c401b4

**yachtsocial.aragonid.eth**

- ENS ([manifest](https://ipfs.eth.aragon.network/ipfs/QmavcfeWqYJ793wZ5UrjuN6DudvRJJSxX167KMvNG39qHD/manifest.json), [artifact](https://ipfs.eth.aragon.network/ipfs/QmavcfeWqYJ793wZ5UrjuN6DudvRJJSxX167KMvNG39qHD/artifact.json)): 0x885912e005098430e0a6d426fe6f0439e48e0bd3
- Compound ([manifest](https://ipfs.eth.aragon.network/ipfs/QmPdUFZDSttmfQUPbDhyQwxepEtwdKs6bF8h4c4DxuKqhg/manifest.json), [artifact](https://ipfs.eth.aragon.network/ipfs/QmPdUFZDSttmfQUPbDhyQwxepEtwdKs6bF8h4c4DxuKqhg/artifact.json)): 0x024cf610eac2ff6f47b677e7eb72f4eb462107b0
- Uniswap ([manifest](https://ipfs.eth.aragon.network/ipfs/QmWkaJH6Xg2qPFnHb4b2QTugVFB16nXhYVfCNMQ37G6F5P/manifest.json), [artifact](https://ipfs.eth.aragon.network/ipfs/QmWkaJH6Xg2qPFnHb4b2QTugVFB16nXhYVfCNMQ37G6F5P/artifact.json)): 0x7deb252158a37889851792a66d59eecda7b935f8

**Comment by lkngtn** [comment]
I'm also noticing some obvious inconsistencies between apps and org templates. There are 65 Open Enterprise deployments reported but only ~45 of the constituent apps reported.

**Comment by onbjerg** [comment]
>I'm also noticing some obvious inconsistencies between apps and org templates. There are 65 Open Enterprise deployments reported but only ~45 of the constituent apps reported.

As we talked about on Keybase this might be due to failed/unfinished deployments for some templates. I'll investigate to make sure though.

**Comment by onbjerg** [comment]
After investigating I was able to re-index all of the apps, except for the ones that did not have a manifest available anymore (which are probably unused), which amounts to 5 unknown apps.

As for the rest, it seems like what happened was probably that we got the version publication event *before* the content was available on IPFS. I think the way to solve this is probably to have a worker that runs in the background every 30 minutes or so that checks if any apps have incomplete information, and if so, tries to fetch the information from IPFS again. It will need a bit more thought before implementing though, as new versions of apps might also suffer from this, and such a worker would not be able to catch that.
